consumer/nnrf.go

```diff
@@ -325,9 +329,9 @@ func SendNFDiscoveryServingAMF(smContext *smf_context.SMContext) (*models.Proble
 	var localErr error
 
 	if smf_context.SMF_Self().EnableNrfCaching {
-		result, localErr = nrfCache.SearchNFInstances(smf_context.SMF_Self().NrfUri, models.NfType_AMF, models.NfType_SMF, &localVarOptionals)
+		result, localErr = nrfCache.SearchNFInstances(context.Background(), smf_context.SMF_Self().NrfUri, models.NfType_AMF, models.NfType_SMF, &localVarOptionals)
 	} else {
-		result, localErr = SendNrfForNfInstance(smf_context.SMF_Self().NrfUri, models.NfType_AMF, models.NfType_SMF, &localVarOptionals)
+		result, localErr = SendNrfForNfInstance(context.Background(), smf_context.SMF_Self().NrfUri, models.NfType_AMF, models.NfType_SMF, &localVarOptionals)
```

Comment:
Same change as the two changes above
```suggestion
	ctx := context.Background()
	if smf_context.SMF_Self().EnableNrfCaching {
		result, localErr = nrfCache.SearchNFInstances(ctx, smf_context.SMF_Self().NrfUri, models.NfType_AMF, models.NfType_SMF, &localVarOptionals)
	} else {
		result, localErr = SendNrfForNfInstance(ctx, smf_context.SMF_Self().NrfUri, models.NfType_AMF, models.NfType_SMF, &localVarOptionals)
```